sample-lnd.conf

```diff
@@ -406,6 +406,12 @@
 ; can be tuned to save bandwidth for light clients or routing nodes. (default: 3)
 ; numgraphsyncpeers=9
 
+; If true, lnd will enable the Prometheus exporter.
```

Comment:
Should mention these flags are behind a build/compile flag and are not available by default. You'll need to use `make install tags="... prometheus"` to activate them.